k8s/perftest/common/draft-store/draft-store-service.yaml

```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: draft-store-service
+  namespace: rpe
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: draft-store-service
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: draft-store-service
+    version: 1.0.4
+  values:
+    rollback:
+      enable: true
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "88e9f6bc-d5ca-4bae-b408-03c271caa4a0"
```

Comment:
@adusumillipraveen should we move this to vault instead? or not worth it?  can't remember where we got to with this

Reply:
I think its worth it to move now, we will be setting up a bad example to others i guess.